server/endpoints_test.go

```diff
@@ -81,3 +81,17 @@ func TestHTTPServer_ProxyTo_VoteEndpointsExtractUserID(t *testing.T) {
 }
 
 // TODO: test that proxying is done correctly including request / response modifiers for all endpoints
+
+func TestHTTPServer_ProxyTo_VoteEndpointBadCharacter(t *testing.T) {
+	badClusterName := "00000000000000000000000000000000000%1F"
```

Comment:
I wish we have the same fuzzer in unit tests which I use in integration tests :)